### DISCUSSION TITLE
Is it bad to have many images of one scene?

### DISCUSSION THREAD

**Discussion by ghost** [ultralytics/yolov5#8987] [Q&A] [answered]
Hello, I saw some persons on YouTube who made videos, split them into frames and labeled each object. I thought, if you have many images of a single scene and split them into train and validation, it could be bad because the validation runs over similar images used in training. Also, nobody uses all images for training and testing and copies a few training images to a validation set. Is that right?

**Comment by glenn-jocher** [comment, marked as answer]
@fennecfly yes this is poor practice because your train and val data is not independent, you will get an artificially high val mAP.

<img width="2029" alt="dataset" src="https://user-images.githubusercontent.com/26833433/185259685-95d0983e-c348-4480-a87a-4ccc325cb301.png">
